test/e2e/far_e2e_test.go

```diff
@@ -277,3 +310,18 @@ func wasNodeRebooted(nodeName string, nodeBootTimeBefore time.Time) {
 
 	log.Info("successful reboot", "node", nodeName, "offset between last boot", nodeBootTimeAfter.Sub(nodeBootTimeBefore), "new boot time", nodeBootTimeAfter)
 }
+
+// remediateNode run three functions to verify whether the node was remediated
```

Comment:
comments are great, but what's the value for saying that we run three functions? Wouldn't it be more useful to say what they are verifying?